models/user.js

```diff
@@ -1,13 +1,26 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
+
+const recipeSchema = new Schema({
+
+	id: mongoose.Schema.Types.ObjectId,
+	ingredients: { type:[String], required: true },
+	description: { type: String, required: true},
+	
+});
+
```

Comment:
Not sure if I should separate the recipe schema into it's own js file since it's fairly small and from what I can see in the mockups recipes are always tied to users(chefs) in some way